metrics_ci_integration/test/test_utils/api_mock_server/request_handler.dart

```diff
@@ -0,0 +1,100 @@
+import 'dart:io';
+
+/// Handle [HttpRequest] callback definition
+typedef RequestHandleDispatcher = Future<void> Function(HttpRequest);
```

Comment:
We should try to reuse api_mock_server from guardian here. We can follow the same approach that we took for yaml - moving api_mock_server to the top level for reuse.